src/Engine/Manager/SoundManager.cpp

```diff
@@ -126,7 +126,9 @@ void SoundManager::Update(float deltaTime) {
         processedSamples = new float[numSamples] {0};
 
     Pa_WriteStream(stream, processedSamples, *numProcessedSamples);
-    delete[] processedSamples;
+
+    if(*numProcessedSamples != 0)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') keyword 'if' not followed by a single space [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT008)